components/KiboAppHeader.vue

```diff
@@ -1,128 +1,176 @@
-              :result="searchSuggestionResult"
+          <div style="flex: 0.35"></div>
+          <div v-click-outside="closeSearch" class="kibo-header__search-bar">
+            <KiboSearchBar
+              ref="searchBarRef"
+              :placeholder="$t('Search for items')"
+              aria-label="Search"
+              class="sf-header__search"
+              :value="term"
               :loading="loading"
-              @closeDialog="closeSearch()"
+              @input="handleSearch"
+              @keydown.enter="gotoSearchResult()"
+              @keydown.esc="closeSearch"
             >
-            </KiboSearchSuggestion>
+              <template #icon>
+                <SfButton
+                  v-if="!!term"
+                  class="sf-search-bar__button sf-button--pure"
+                  @click="closeOrFocusSearchBar"
+                >
+                  <span class="sf-search-bar__icon">
+                    <SfIcon color="var(--c-text)" size="18px" icon="cross" />
+                  </span>
+                </SfButton>
+                <SfButton
+                  v-else
+                  class="sf-search-bar__button sf-button--pure"
+                  @click="isSearchOpen ? (isSearchOpen = false) : (isSearchOpen = true)"
+                >
+                  <span class="sf-search-bar__icon">
+                    <SfIcon color="var(--c-text)" size="20px" icon="search" />
+                  </span>
+                </SfButton>
+              </template>
+            </KiboSearchBar>
+            <div class="search-suggestion-div">
+              <KiboSearchSuggestion
+                :visible="isSearchOpen"
+                :result="searchSuggestionResult"
+                :loading="loading"
+                @closeDialog="closeSearch()"
+              >
+              </KiboSearchSuggestion>
+            </div>
+          </div>
+
+          <div v-if="!isMobile" class="sf-header__icons">
+            <SfButton
+              v-e2e="'app-header-location'"
+              class="sf-button--pure sf-header__action"
+              @click="handleStoreLocatorClick"
+            >
+              <SfIcon class="sf-header__icon">
+                <font-awesome-icon icon="map-marker-alt" class="fa-icon" />
+              </SfIcon>
+              <div class="kibo-header__icon">
+                <span class="kibo-header__icon-name"> {{ selectedLocation }}</span>
+              </div>
+            </SfButton>
+            <SfButton
+              v-e2e="'app-header-account'"
+              class="sf-button--pure sf-header__action"
+              @click="handleAccountClick"
+            >
+              <SfIcon class="sf-header__icon">
+                <font-awesome-icon :icon="[accountIcon, 'user-circle']" class="fa-icon" />
+                <i class="far fa-user-circle"></i>
+              </SfIcon>
+              <div class="kibo-header__icon">
+                <span class="kibo-header__icon-name"> My Account</span>
```

Comment:
Add this to en.js or import if exists

Reply:
Resolved in PR - https://github.com/KiboSoftware/nuxt-storefront/pull/138